Dockerfile.pr

```diff
@@ -0,0 +1,70 @@
+# Form viewier that runs as a Lambda function.  It is used to create 
+# Lambda function review environments for each PR.
+FROM node:16@sha256:2ed6bba040f90005db9785927689c0d9a9442ca2cf9a59dc52297d684285c094
+
+ENV NODE_ENV=production
+
+COPY . /src
+WORKDIR /src
+
+ARG GOOGLE_CLIENT_ID
+ARG GOOGLE_CLIENT_SECRET
+ARG COGNITO_REGION="ca-central-1"
+ARG COGNITO_APP_CLIENT_ID
+ARG COGNITO_USER_POOL_ID
+
+RUN yarn install --silent --production=false
+RUN yarn build
+RUN yarn install --production
+
+FROM node:16@sha256:2ed6bba040f90005db9785927689c0d9a9442ca2cf9a59dc52297d684285c094
+
+COPY flag_initialization /src
+WORKDIR /src
+RUN yarn install --silent 
+
+FROM node:16@sha256:2ed6bba040f90005db9785927689c0d9a9442ca2cf9a59dc52297d684285c094
+LABEL maintainer="-"
+
+ARG GOOGLE_CLIENT_ID
+ENV GOOGLE_CLIENT_ID=$GOOGLE_CLIENT_ID
+
+ARG GOOGLE_CLIENT_SECRET
+ENV GOOGLE_CLIENT_SECRET=$GOOGLE_CLIENT_SECRET
+
+ARG GITHUB_SHA_ARG
+ENV GITHUB_SHA=$GITHUB_SHA_ARG
+
+ARG COGNITO_REGION="ca-central-1"
+ENV COGNITO_REGION=$COGNITO_REGION
+
+ARG COGNITO_APP_CLIENT_ID
+ENV COGNITO_APP_CLIENT_ID=$COGNITO_APP_CLIENT_ID
+
+ARG COGNITO_USER_POOL_ID
+ENV COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID
+
+ARG TAG_VERSION
+ENV TAG_VERSION=$TAG_VERSION
+
+WORKDIR /src
+
+COPY package.json yarn.lock ./
+
+COPY --from=0 /src/node_modules ./node_modules
+COPY --from=0 /src/.next ./.next
+COPY public ./public
+COPY next.config.js .
+COPY next-i18next.config.js .
+COPY prisma ./prisma
```

Comment:
@bryan-robitaille is the `prisma` folder only needed for DB migrations?  If yes, I'll remove it from the Dockerfile for even more certainty that a PR review env cannot accidentally migrate the DB.

Reply:
Unfortunately the prisma folder is required for the app the run.  Prisma requires the folder to create an instance of itself on startup so that the Next Js app can communicate with the database.